src/main/java/pl/jablonskanycz/bakery/BakeryApplication.java

```diff
@@ -37,8 +37,8 @@ public static void main(String[] args) {
         addressService.updateAddress(1L, 16.90, 18.30);
         System.out.println(addressService.getAllAddresses());
         System.out.println("Delete address");
-//        addressService.deleteAddress(1L);
-//        System.out.println(addressService.getAllAddresses());
+        addressService.deleteAddress(8L);
```

Comment:
why number `8` :)? Is it important that it is `8` or could be any other number? _any_ number or perhaps a number that ... (fulfills some requirements in the app)?

Reply:
It's just for tests reason. I wanted to pass here an argument obtained from a Client but I don't have ClientRepository and Service yet. Maybe it's not necessary but for now I'm not aware of it :)

Reply:
I see. When you are running your app, does this `8` work or does it throw an exception :)?

Reply:
It works :)

Reply:
I see. So we now have a state where we want to test a deletion of an address that had existed before deletion. However, the chosen id (`8`) seems not to be working on _any_  environment (since it doesn't work for me but it does for you). The best option would be to make it work on all computers so that anyone who will pull your repository will pull a working setup. 
 Let's come up with some actions that would help you reproduce the issue too.

The first step would be for us to make sure we on the same database state. What I did was I pulled your latest changes, deleted whatever image/container I had before and did docker build and run as a fresh start. Could you check if following these steps makes the issue come or would it still not-be-an-issue on your computer?

Reply:
I did what you suggested and it's not working any more :( I even added  a logging to better trace what's going on there. So the method delete from the addressRepository does not execute deletion. What to do now?

Reply:
good spot! 
Can you check please if it worked before the proposed changes? :) 
Can you please check using `id = 1` ?

Reply:
After docker build and run it stopped working. I checked it with every id and it deleted only id=5 which was the last element in address table.

Reply:
yup. 
I can give you the answer up-front what needs to be corrected however, I would prefer to rather give you hints first so that you can learn how to solve such problems on your own in the future :) 

Therefore, let me ask you these questions:
Since we know Spring doesn't allow us to successfully delete the address with `id < 5`, let's check if other program would allow us - `DBeaver`. Such program like `Dbeaver` are only tools to run SQL against the database so if we can successfully delete an address there we will know the problem lies in Java/Spring. But if we still cannot do it there, we will then take a look at the sql and database. Therefore: 
Please run corresponding query in there and take a look at the result